Segment_Delaunay_graph_2/test/Segment_Delaunay_graph_2/test_sdg_2_et.cpp

```diff
@@ -1,54 +1,17 @@
-#include <CGAL/basic.h>
```

Comment:
This commit is responsible for the compilation errors in:
https://cgal.geometryfactory.com/CGAL/testsuite/results-5.0-Ic-26.shtml#Segment_Delaunay_graph_2

Reply:
We should test it again. I want to check that the error only happens in the draw functions, and if so fix that after a successfull testsuite without calling draw functions.

Reply:
What gave the tests on your machine?

Reply:
On my Linux machine, after commits https://github.com/CGAL/cgal/pull/3797/commits/c738d89acaf68f0dcd732f269d296744d73bee84 and https://github.com/CGAL/cgal/pull/3797/commits/7ffbf131bb615e19699680daa2e739ffb123d066, the following tests passed:
```shell-session
$ make && ctest -R 'test_sdg_2_et|test_sdg_hierarchy_2_et' -j2
    Start 3812: compilation_of__test_sdg_2_et
    Start 3814: Segment_Delaunay_graph_2_Tests_SetupFixture
1/6 Test #3814: Segment_Delaunay_graph_2_Tests_SetupFixture .....   Passed    0.01 sec
    Start 3818: compilation_of__test_sdg_hierarchy_2_et
2/6 Test #3818: compilation_of__test_sdg_hierarchy_2_et .........   Passed   15.22 sec
    Start 3819: execution___of__test_sdg_hierarchy_2_et
3/6 Test #3812: compilation_of__test_sdg_2_et ...................   Passed   15.82 sec
    Start 3813: execution___of__test_sdg_2_et
4/6 Test #3819: execution___of__test_sdg_hierarchy_2_et .........   Passed    0.83 sec
5/6 Test #3813: execution___of__test_sdg_2_et ...................   Passed    1.11 sec
    Start 3815: Segment_Delaunay_graph_2_Tests_CleanupFixture
6/6 Test #3815: Segment_Delaunay_graph_2_Tests_CleanupFixture ...   Passed    0.01 sec

100% tests passed, 0 tests failed out of 6

Label Time Summary:
Segment_Delaunay_graph_2_Tests    =  33.00 sec*proc (6 tests)

Total Test time (real) =  17.10 sec
30.40s user 2.40s system 191% cpu 17.109s total
```